Header header
MotorError errorStatus
SerialError serialError
LimitStatus limitStatus
int16 targetSpeed
int16 speed
int16 brakeAmt
int16 vin
int16 temp
